lang/ui.en.json

```
@@ -994,5 +1018,17 @@
   "unknown": {
     "defaultMessage": "unknown",
     "description": "Label for unknown ML event"
+  },
+  "unplug-radio-link-microbit-description": {
+    "defaultMessage": "Unplug the radio link micro:bit from the computer. If you want to record more data samples, you will need to reconnect with the tool.",
```

Maybe use consistent copy with the BT scenario?

```suggestion
    "defaultMessage": "Unplug the radio link micro:bit from the computer. You will need to reconnect the micro:bit if you want to record more data samples.",
```